coalib/coala.py

```diff
@@ -55,8 +56,22 @@ def main(debug=False):
 
         if args.show_bears:
             from coalib.settings.ConfigurationGathering import (
-                get_all_bears)
-            filtered_bears = get_all_bears()
+                get_all_bears, load_configuration)
+            from coalib.collecting.Collectors import collect_bears
+            if args.bears:
+                sections, _ = load_configuration(arg_list=None)
+                local_bears = {}
+                global_bears = {}
+                for (key, value) in sections:
+                    bear_dirs = value.bear_dirs()
+                    local_bears[key], global_bears[key] = collect_bears(
+                        bear_dirs,
+                        args.bears,
+                        [BEAR_KIND.LOCAL, BEAR_KIND.GLOBAL],
+                        warn_if_unused_glob=False)
```

Comment:
Take a look at [collect_all_bears_from_sections](https://github.com/coala/coala/blob/master/coalib/collecting/Collectors.py#L292-L311)

Basically you are just duplicating this code.
I think it would be more beneficial to refactor that function (or some other, related function) to take a bear-glob parameter. This would also help to keep code related to bear collecting in the Collectors.py module.